### PR TITLE
fe: use constant bonder fee last digit

### PR DESCRIPTION
**Pull request by shanefontaine**
To review:

* Is this the correct place to put this logic?

### CODE REVIEW

File: packages/frontend/src/utils/modifyBonderFee.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { BigNumber } from 'ethers'
+
+function modifyBonderFee (bonderFee: BigNumber, modifyAmount: number): BigNumber {
```
[the remark this answers is not in the document]
Comment (shanefontaine) [reply packages/frontend/src/utils/modifyBonderFee.ts]: Agree with all. Updates 3af806e2d198a1f408b33d774e9f7d26b6f6a39d.

File: packages/frontend/src/utils/modifyBonderFee.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  const feeStr: string = bonderFee.toString()
+  const modifiedFee: string = feeStr.substring(0, feeStr.length - 1) + modifyAmount.toString()
```
[the remark this answers is not in the document]
Comment (shanefontaine) [reply packages/frontend/src/utils/modifyBonderFee.ts]: Agree. I generalized it more with 3af806e2d198a1f408b33d774e9f7d26b6f6a39d.

**Comment by shanefontaine** [comment]
@miguelmota back to you

**Comment by miguelmota** [comment]
@shanefontaine changes look great. Ready for merge

**Pull request merged**